### PR TITLE
SDI-213: ♻️  Switch to new prisoners unaccounted for endpoint

### PR DESCRIPTION
**Pull request by petergphillips**
It still won't be speedy since there are issues with the Prison API endpoint.  Should be a bit better though as only making one call to get the data and also new 30 second timeout on the endpoint.

### CODE REVIEW

**Review by Mjwillis — APPROVED** [review]
LGTM

File: backend/api/prisonApi.ts [hunk 1]
```diff
@@ -274,9 +274,6 @@ export const prisonApiFactory = (client) => {
   const addSingleAppointment = (context, bookingId, body) =>
     post(context, `/api/bookings/${bookingId}/appointments`, body)
 
-  const getOffenderActivities = (context, { agencyId, date, period }) =>
```
Comment (petergphillips) [inline backend/api/prisonApi.ts:277]: removed - can now call whereabouts to get that info

File: backend/api/whereaboutsApi.ts [hunk 2]
```diff
@@ -37,8 +37,10 @@ export const whereaboutsApiFactory = (client) => {
 
   const getAbsenceReasons = (context) => get(context, '/absence-reasons')
 
-  const getPrisonAttendance = (context, { agencyId, date, period }) =>
-    get(context, `/attendances/${agencyId}/attendance-for-scheduled-activities?date=${date}&period=${period}`)
+  const prisonersUnaccountedFor = (context, { agencyId, date, period }) =>
```
Comment (petergphillips) [inline backend/api/whereaboutsApi.ts:40]: new endpoint with increased timeout as can take up to 30 seconds

**Review by Mjwillis — APPROVED** [review]
LGTM

**Pull request merged**